### PR TITLE
Hw 15

### PR DESCRIPTION
**Pull request by Kaijou88**
…h webapp

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: mfran
+  Date: 4/27/2020
+  Time: 12:54 PM
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/WEB-INF/views/index.jsp]: let's remove comments created by idea

File: src/main/webapp/web.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <servlet-name>index</servlet-name>
+        <servlet-class>mate.academy.internetshop.controllers.IndexController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+
+</web-app>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]: add empty line to the end of file

**Review by dmytro-hryhoruk — COMMENTED** [review]
Let's set up travis ci to check your next PR's https://www.youtube.com/watch?v=BHSsWTbuow4

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 1 above)
Comment (Kaijou88) [reply src/main/webapp/WEB-INF/views/index.jsp]: deleted

File: src/main/webapp/web.xml (hunk 2 above)
Comment (Kaijou88) [reply src/main/webapp/web.xml]: added

**Comment by Kaijou88** [comment]
set up travis. thank you

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - mvn clean install
+cache:
+  directories:
+    - $HOME/.m2
```
Comment (dmytro-hryhoruk) [inline .travis.yml]: add empty line

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**